src/mods/common/analyzer/append/mathutils.mod.rst

```diff
@@ -6,6 +6,14 @@
 
    .. method:: __init__(rgb=(0.0, 0.0, 0.0))
 
+   .. method:: __get__(instance, owner)
+
+      :rtype: :class:`Color`
+
+   .. method:: __set__(instance, value)
+
+      :type value: :class:`Color`, collections.abc.Sequence[float]
```

Comment:
If you use `collections.abc.Sequence[float]` is the mod file, you need to add `skip-refine` option.
Otherwise, `collections.abc.Sequence[float]` will be skipped.

I think `Xd vector` or just `:class:`Color` is preferable which will be refined (`collections.abc.Sequence[float]` is added) by DataTypeRefiner.

```suggestion
      :type value: :class:`Color`
```

Same for others.

Reply:
I removed `Sequence` and let DataTypeRefiner add it.